.openpublishing.redirection.json

```diff
@@ -294,6 +294,10 @@
         "redirect_url": "/visualstudio/ide/visual-studio-ide"
     },
     {
+        "source_path": "docs/visual-basic/getting-started/breaking-changes-in-visual-studio-2015.md", 
```

Comment:
not so sure about this one. I think we should keep the topic but remove version from the file name and make the topic cumulative over time. I'll redirect all the C# and VB topics from VS.140 (Vs 2015) version to here soon.